### ISSUE TITLE
Accessibility in Warning Callout label - Failing Axe checks due to invalid aria role

### ISSUE THREAD

**Issue by Tomdango** [nhsuk/nhsuk-frontend#817]
# Bug Report

## What is the issue?

The Warning Callout label has an invalid aria role in the label, causing an Axe failure.

## What steps are required to reproduce the issue?

Running Axe over the following HTML:

```html
<h2 class="nhsuk-warning-callout__label">
    <span role="text"><span class="nhsuk-u-visually-hidden">Important: </span>Data Quality</span>
</h2>
```

Produces the following error:

```
    Expected the HTML found at $('span[role="text"]') to have no violations:

    <span role="text"><span class="nhsuk-u-visually-hidden">Important: </span>Data Quality</span>

    Received:

    "ARIA roles used must conform to valid values (aria-roles)"

    Fix all of the following:
      Role must be one of the valid ARIA roles: text

    You can find more information on this issue here: 
    https://dequeuniversity.com/rules/axe/3.5/aria-roles?application=axeAPI
```

The `text` role does not exist in the W3C aria spec: https://www.w3.org/TR/html-aria/
I'd imagine the fix would be to simply remove the offending `<span role="text">` element, and would look like the following:

```html
<h2 class="nhsuk-warning-callout__label">
    <span class="nhsuk-u-visually-hidden">Important: </span>Data Quality
</h2>
```

## What was the environment where this issue occurred?

* Device:  N/A (Running in a CI environment)
* Operating System: N/A
* Browser: N/A
* Browser version: N/A
* NHS.UK frontend package version: 4.1.0

The `nhsuk-react-components` library is being used here, but the issue exists in both packages, in the Nunjucks template and in the service manual website.

**Comment by andymantell** [comment]
I imagine it'll be something to do with this:

https://dequeuniversity.com/rules/axe/4.3/aria-text

> When a text node is split by markup (e.g. <h1>Hello <span>World</span></h1>) VoiceOver will treat it as two separate phrases instead of just one. Adding role="text" around the elements solves the problem.

**Comment by wa-rren-dev** [comment]
Good call @andymantell, I'll try and dig into this a bit

**Comment by wa-rren-dev** [comment]
I've tested this and it is indeed still an issue on VoiceOver for iOS. My opinion would be that we leave it in to prioritise usability over issues flagged by automated testing. @mcheung-nhs could I get your take?

**Comment by sambrown3-nhs** [comment]
If it helps at all, @Tomdango and myself are no longer seeing this issue after updating the version of `aXe` we're using for automated testing.

**Comment by wa-rren-dev** [comment]
> If it helps at all, @Tomdango and myself are no longer seeing this issue after updating the version of `aXe` we're using for automated testing.

Brill, thanks for letting me know. For future, do you know what versions you've moved from and to?

**Comment by sambrown3-nhs** [comment]
We're using the `jest-axe` wrapper, and we've moved from `3.4.0` to `6.0.0`, in terms of `axe-core`, it looks to be a change from `3.5.?` to `4.4.1`
